.travis.yml

```diff
@@ -4,7 +4,10 @@ rvm:
   - jruby-19mode
   - 1.9.3
   - 2.0.0
-  - 2.1.0
```

Comment:
@nettofarah @rohitpaulk curious if you have insights into how we we should pick the versions to support going forwards. Should we just do whatever we happen to support - or are their some adoption numbers we can use as guidelines (e.g. for Android we support any version > 5% on https://developer.android.com/about/dashboards/index.html).

Reply:
Couldn't find a similar resource for Ruby, but stumbled across the following: 

- https://github.com/rubygems/rubygems.org/issues/1335
- https://blog.jelastic.com/2016/04/14/software-stacks-market-share-first-quarter-of-2016/ - says that 1.9 usage accounted for < 6% in 2016. I could't find details on where they got the data from though.
- https://semaphoreci.com/blog/2014/08/27/ruby-version-usage-in-commercial-projects-2014-edition.html - says that 1.9 usage accounted for ~50% in 2014, based on usage from private projects on SemaphoreCI.

As far as I've observed though, a lot of popular client libraries have dropped support for ruby < 2.0. Examples: 

[Octokit](https://github.com/octokit/octokit.rb) (github API client), [Stripe](https://github.com/stripe/stripe-ruby), [Twilio](https://github.com/twilio/twilio-ruby).

Examples of client libraries that still support 1.9:

[Raven](https://github.com/getsentry/raven-ruby/blob/master/sentry-raven.gemspec) (Sentry client),

Reply:
Gotcha. Unfortunately we also don't have our own internal numbers to be able to make a decision here yet.

Based on what other libraries have been doing, it does seem to me that once we push out 2.1.0 of analytics-ruby and clean up some of the existing bugs, we might be ok to drop 1.9 Ruby support if we need to use newer APIs.